client/views/session.tpl

```diff
@@ -0,0 +1,15 @@
+<p class="clearfix">
+{{#if current}}
+	<strong>{{agent}}</strong>
+	<a href="https://ipinfo.io/{{ip}}" target="_blank" rel="noopener">{{ip}}</a>
+	<br>
+	{{localetime lastUse}}
```

Comment:
You can remove this one, it will just always give the current time or something like that lol, not super useful I think.